change/@ni-nimble-components-37c84290-c5e8-4c31-a22e-e69547cc96c0.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I may have misheard, but I thought Brandon had mentioned that he wanted tabs to receive the new semi-bold font. In Chromatic I'm seeing that tab toolbar buttons got that change but the titles of tabs are still regular font. Can you double check what's supposed to happen with tabs?

Reply:
Yup, the Figma spec shows the tabs as semi-bold font. it'll show as updated in my next commit.